testsuite/forge.py

```diff
@@ -1320,8 +1282,8 @@ def test(
                 shell,
                 git,
                 2,
-                enable_failpoints_feature=forge_enable_failpoints,
-                enable_performance_profile=forge_enable_performance,
+                enable_failpoints_feature=forge_enable_failpoints == "true",
```

Comment:
Either keep this, or keep the stuff on line 1275-1276:

```
forge_enable_failpoints = forge_enable_failpoints == "true"
forge_enable_performance = forge_enable_performance == "true"
```

Reply:
1275-76 is just aws stuff, or did you mean move this to that line?

Reply:
Ah I see in the rebased version